apisix/plugins/jwt-auth.lua

```diff
@@ -18,8 +18,9 @@ local core     = require("apisix.core")
 local jwt      = require("resty.jwt")
 local consumer_mod = require("apisix.consumer")
 local resty_random = require("resty.random")
-local vault        = require("apisix.core.vault")
+local vault        = require("apisix.core.vault") -- to be removed later
 local new_tab = require ("table.new")
+local secrets = require("apisix.admin.secrets")
```

Comment:
All functions in `apisix.admin.secrets` are only used to configure secret manager. It should not be called from plugins.

If you want to use the secret manager function, you should use the `get` function in `apisix.secret`